tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/run_direct_ptx_fused_linear_competitors.py

```
@@ -0,0 +1,157 @@
+    device = torch.device("cuda")
+    torch.set_grad_enabled(False)
+    for run in range(1, args.runs + 1):
+        torch.manual_seed(20261400 + run)
+        for name, input_features, output_features in SHAPES:
+            x = (torch.rand((input_features,), device=device) * 2.0 - 1.0) * 0.125
+            weights = (
+                torch.rand((output_features, input_features), device=device) * 2.0 - 1.0
+            ) * 0.0625
+            bias = (torch.rand((output_features,), device=device) * 2.0 - 1.0) * 0.0625
+
+            def operation():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `operation`

(ANN202)